### DISCUSSION TITLE
Docker

### DISCUSSION THREAD

**Discussion by sairuk** [retronas/discussions#112] [Ideas]
All docker discussion will be moved here from the main RetroNAS project.

See: 
- [Previous discussion](https://github.com/danmons/retronas/discussions/2)
- [docker-aio](https://github.com/danmons/retronas-dist/tree/docker-aio) branch
- A hacked up integration with the [current project](https://github.com/danmons/retronas/tree/docker-example/ansible/templates/build_docker)
- a demo from @willoucom https://github.com/danmons/retronas/pull/10

**Comment by EmSixTeen** [comment]
Just commenting in here to voice my interest in this, as I don't want anyone to think no-one is! Still getting my head around Docker for some other things, so can't contribute myself.
